pkg/testutil/nerdtest/requirements.go

```diff
@@ -19,125 +19,293 @@ package nerdtest
 import (
 	"encoding/json"
 	"fmt"
-	"testing"
+	"os"
+	"os/exec"
+	"strings"
 
 	"gotest.tools/v3/assert"
 
 	"github.com/containerd/nerdctl/v2/pkg/buildkitutil"
 	"github.com/containerd/nerdctl/v2/pkg/inspecttypes/dockercompat"
 	"github.com/containerd/nerdctl/v2/pkg/rootlessutil"
-	"github.com/containerd/nerdctl/v2/pkg/testutil"
+	"github.com/containerd/nerdctl/v2/pkg/testutil/nerdtest/platform"
 	"github.com/containerd/nerdctl/v2/pkg/testutil/test"
 )
 
+var BuildkitHost test.ConfigKey = "bkHost"
```

Comment:
Should consistently begin with an upper char?

Reply:
Some/most of these config keys are internal only and not meant to be used outside of the package (they are set by `Requirements`).

This one is an exception (for now) as I have not touched much the buildkit stuff - and for the time being it needs to be exported outside of the package.

Will definitely revisit in the next PR when I will do the `builder` tests.

Reply:
Now I am wondering if I understood what you asked.

Were you instead suggesting the value to be "BkHost"?

I will adjust - actually, will adjust to "BuildkitHost".